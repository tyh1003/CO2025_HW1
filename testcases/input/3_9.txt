10
2 19 71 22 14 99 32 34 66 97
