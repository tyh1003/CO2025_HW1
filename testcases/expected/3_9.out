2 14 19 22 32 34 66 71 97 99 
